docs/source/references/modules.rst

```diff
@@ -97,10 +97,10 @@ SI_SDR
 .. autoclass:: torchmetrics.SI_SDR
     :noindex:
 
-SI_SNR
-~~~~~~
+ScaleInvariantSNR
```

Comment:
This introduces a new inconsistency since in the functional part you don't abbreviate snr, let's also be consistent here

Reply:
to be clear you suggest naming it `ScaleInvariantSignalNoiseRatio`, correct?

Reply:
yes either that or abbreviating snr in the functional interface as well :)

Reply:
on one hand, we have `MeanAbsolutePercentageError` instead of just MAPE but then we have `ROC` instead of  Receiver Operating Characteristic

Reply:
Cannot we have full name in all metrics levels and have full name and abbreviate name in each small field?

Reply:
I think the expanded names are beneficial in many cases. However, for example, `BLEU = bilingual evaluation understudy` or `ROUGE =  Recall-Oriented Understudy for Gisting Evaluation` would be really confusing.

Reply:
@justusschock I found another interesting mix `pit_permutate`

Reply:
I think that unrolling retrieval names will just add complexity...
- `RetrievalMAP` -> `RetrievalMeanAveragePrecision`
- `RetrievalMRR` -> `RetrievalMeanAveragePrecision`
- `RetrievalNDCG` -> `RetrievalNormalizedDiscountedCumulativeGain`

Reply:
@quancs  if I am not wrong SDR and SNR are general metrics, so not only for audio, correct?
if so can your implementation can be used also for Image...? :rabbit:

Reply:
Maybe yes. Images doesn't have the time dim. If that is a problem, maybe we can remove SDR and SNR from the top level.

Reply:
I do not mind keeping it in top-level, rather thinking about `AudioSDR` and `AudioSND`?

Reply:
If let me choose, I would prefer remove SDR and SNR from the top level